libraries/hardwareInterfaces.js

```diff
@@ -328,6 +328,64 @@ exports.triggerUDPCallbacks = function(msgContent) {
     });
 };
 
+exports.clearTool = function(objectName, frameName){
+    var objectID = utilities.getObjectIdFromTarget(objectName, objectsPath);
+    console.log('remove set tool');
+
+    var frameUuid = objectID + frameName;
+
+    if (!_.isUndefined(objectID) && !_.isNull(objectID)) {
+        if (objects.hasOwnProperty(objectID)) {
```

Comment:
These two lines could be simplified as just `if (objects.hasOwnProperty(objectID)) {` since `objects` will never contain a null or undefined objectID